maint.php

```diff
@@ -472,9 +477,9 @@ function get_header_label() {
 			FROM plugin_maint_schedules
 			WHERE id = ?',
 			array(get_filter_request_var('id')));
-		$header_label = __esc('General Settings [edit: %s]', $list['name']);
+		$header_label = __('[edit: %s]', $list['name'], 'maint');
```

Comment:
Doing this could lead to XSS I believe as it removes the escaping around the values.  You should keep the `__esc` function call.  Was there a reason to change it?  I noticed you introduced htmlspecialchars further down.

Reply:
Function htmlspecialchar was added back per the first feedback.  The escaping is added using htmlspecialchar on $header_label when the actual page is generated.

__esc was removed due to double escaping.  A schedule with name "<script>al</script>" showed on the webpage as "&lt;script&gt;al&lt;/script&gt;".  The double escaping only affects existing schedules with HTML (mine was a test before preventing it) in the schedule name.  The initial commit includes a change to remove values "<>" from new/changed schedule names so double escaping does not matter.

"__esc" is added back.